### PR TITLE
Test: add unit testing for workflow-service execution

### PR DESCRIPTION
**Pull request by eloycoto**

### CODE REVIEW

File: workflow-service/src/test/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WorkFlowServiceImplTest {
+
+	private WorkFlowDefinitionRepository wfDefRepo;
```
Comment (pkliczewski) [inline workflow-service/src/test/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImplTest.java:26]: In similar way we have naming conventions [here](https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names)

**Review by pkliczewski — COMMENTED** [review]
Overall looks good to me.

File: workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java [hunk 2]
```diff
@@ -30,10 +30,9 @@
 import com.redhat.parodos.workflows.work.WorkReport;
 import com.redhat.parodos.workflows.work.WorkStatus;
 import com.redhat.parodos.workflows.workflow.WorkFlow;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+
+import java.util.*;
```
[the remark this answers is not in the document]
Comment (lshannon) [reply workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java:34]: Let's discuss, we might want to soften on that one

**Review by RichardW98 — APPROVED** [review]

**Pull request merged**